### DISCUSSION TITLE
Module '"vue-router/auto"' has no exported member 'createRouter'

### DISCUSSION THREAD

**Discussion by entinio** [posva/unplugin-vue-router#347] [Questions and Help] [answered]
I created today a fresh Vue3 install with vue-router 4.3.0 and unplugin-vue-router 0.8.5 installed.

Changed my vite.config.ts:

```
(...)
import VueRouter from 'unplugin-vue-router/vite'

export default defineConfig({
  plugins: [
    VueRouter({}),
    vue(),
    (...)
  ],
  (...)
})
```

Changed my tsconfig.node.json:

```
{
  (...)
  "include": [
    (...)
    "typed-router.d.ts"
  ],
  "compilerOptions": {
    (...)
    "moduleResolution": "Bundler",
    "types": ["node", "unplugin-vue-router/client"]
  }
}
```

And then, when I changed my import with:

`import { createRouter, createWebHistory } from 'vue-router/auto'`

I directly get this error : Module '"vue-router/auto"' has no exported member 'createRouter'

Since I followed the doc, what did I miss here?

**Comment by entinio** [comment]
I'll put the answer here for anyone : you need to add the config to both tsconfig.node.json and tsconfig.app.json (might need to update the doc for those not used to ts config files like me)

**Comment by Jaciezyt** [comment, marked as answer]
It seems that there is no need to edit tsconfig.node.json
I only configured tsconfig.app.json and it works for me

**Reply by posva** [reply]
It's also worth noting that it's better to a `/// <reference>` comment (https://uvr.esm.is/introduction.html#setup)
